packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -65,7 +65,7 @@ export class AttestationService {
   public start = async (): Promise<void> => {
     const slot = this.provider.getCurrentSlot();
     //trigger getting duties for current epoch
-    this.onNewEpoch(computeEpochAtSlot(this.config, slot) - 1);
+    await this.onNewEpoch(computeEpochAtSlot(this.config, slot) - 1);
```

Comment:
We could use this
```suggestion
    void this.onNewEpoch(computeEpochAtSlot(this.config, slot) - 1);
```
to intentionally ignore result.

See comment:
https://github.com/Microsoft/TypeScript/issues/13376#issuecomment-273289748

Reply:
What's the issue with using `await`? Don't we want to await `onNewEpoch`?

Reply:
Well if attestation is on first slot, start method would wait quite a lot of time to resolve. Errors in those methods ain't suppose to propagate up. it should be handled and this service should just loop and wait for next duty